site/source/docs/compiling/WebAssembly.rst

```diff
@@ -86,6 +87,13 @@ upgrade from fastcomp to upstream:
   around the fact that we can't grow the table. In the upstream backend table
   growth is easy, and you can just enable ``ALLOW_TABLE_GROWTH``.
 
+* Fastcomp and upstream use very different LLVM and clang versions (fastcomp
+  has been stuck on LLVM 6, upstream is many releases after). This affects
+  optimizations, usually by making the upstream version faster and smaller.
+  However, in rare cases you may see a regression (for example, in some cases
+  UNoptimized code may be
```

Comment:
Are `UN` uppercase intentionally?

Reply:
Yeah, because I think it's surprising that this is about a speed difference in unoptimized code, where people normally don't compare speed. Is it not clear enough?

Reply:
No I think it sounds fine.

Reply:
Oh I didn't even notice this. Maybe add italics or something to make the emphasis more clear?

Reply:
Added italics and a dash in addition to the all caps now :wink: Also I added text to remind people to optimize.